packages/components/pie-button/package.json

```diff
@@ -33,6 +33,9 @@
     "@justeattakeaway/pie-css": "workspace:*",
     "@justeattakeaway/pie-webc-core": "workspace:*"
   },
+  "peerDependencies": {
+    "react": "^17.0.0"
```

Comment:
Do we want to define this as a hard peerDep? Mainly as some consumers won't need this, and if not, NPM will auto-install it as it's marked as a peerDependency.